homeassistant/components/scrape/sensor.py

```diff
@@ -164,12 +191,14 @@ def _extract_value(self) -> Any:
 
     async def async_added_to_hass(self) -> None:
         """Ensure the data from the initial update is reflected in the state."""
+        await ManualTriggerEntity.async_added_to_hass(self)
         await super().async_added_to_hass()
```

Comment:
I don't like this, I think it's unclear.
Did mypy complain also in CI, just to make sure it was not an issue locally due to stale mypy cache or something like that?

Reply:
Didn't test but we had to do same here in a previous PR.
https://github.com/home-assistant/core/blob/dev/homeassistant/components/template/trigger_entity.py#L27

Reply:
I set it back with the ignore.
I think that's the best that can be made and also is mostly clear on what's going on.